### ISSUE TITLE
(SyntaxError) <script>:2: syntax error, unexpected tIDENTIFIER

### ISSUE THREAD

**Issue by tkrotoff** [laurilehmijoki/s3_website#140]
I get this when running `s3_website push`:

```
[wait] Failed to upload l'osteopathie/index.html ((SyntaxError) <script>:2: syntax error, unexpected tIDENTIFIER
       File.fnmatch('assets/*', 'l'osteopathie/index.html')
                        ^). Trying again in 1 second.
```

It seems that s3_website does not like filenames that contain character `'`.

**Comment by tkrotoff** [comment]
It seems that [this line of code](https://github.com/laurilehmijoki/s3_website/blob/v2.7.2/src/main/scala/s3/website/model/push.scala#L87) is the only place with a `File.fnmatch`.
Unfortunately I haven't done any Scala in my life.

**Comment by laurilehmijoki** [comment]
Thanks for pointing out this bug, @tkrotoff! I have just released the version **2.7.3**, which fixes the problem.

The fix commit is here: https://github.com/laurilehmijoki/s3_website/commit/06ddfc44b455c1005d1cee9cf7b52d085bfe5ced.

**Comment by tkrotoff** [comment]
Thx for the nice and quick release
